ozaria/site/styles/play/level/tome/problem_alert.sass

```diff
@@ -8,16 +8,26 @@
   // Position these at the end of the spell editor, right above the spell toolbar.
   top: 45px
   right: 500px
-  background: transparent
-  border: 1px solid transparent
-  padding: 0
-  font-size: 18px
+  background-color: #ffffff
+  padding: 18.5px 13px
+  font-size: 16px
+  font-family: "Open Sans"
```

Comment:
Is this something that can be inherited correctly?

Reply:
Can you explain?

Reply:
Resolved offline.